README.md

```diff
@@ -77,7 +77,8 @@ Priority order:
 | storage.maxUploadedFileSize                   |     536870912      |    No    |Maximum size in bytes of uploaded file. If a size of uploaded file exceeds the limit the server returns HTTP code 413
 | encryption.secret                             |         -          |    No    |Secret is used for AES encryption of a prefix to the bucket blob storage. The value should be random generated string.
 | encryption.key                                |         -          |    No    |Key is used for AES encryption of a prefix to the bucket blob storage. The value should be random generated string.
-| resources.maxSize                             |      1048576       |    No    |Max allowed size in bytes for a resource.
+| resources.maxSize                             |      67108864      |    No    |Max allowed size in bytes for a resource.
+| resources.cacheMaxSize                        |      1048576       |    No    |Max size in bytes for a resource to cache in Redis.
```

Comment:
maxCachedResourceSize - max size in bytes of a resource to cache in Redis?
cacheMaxSize has a different meaning to me

Reply:
or maxCachedSize

Reply:
renamed to maxSizeToCache